src/Components/SavedSearchAlert/types.ts

```diff
@@ -63,6 +63,11 @@ export interface SavedSearchAlertFormValues {
   frequency: SavedSearchFrequency
 }
 
+export interface HearFromArtsyAdvisorFormValues {
```

Comment:
I decided to separate these values rather than add them to the existing `SavedSearchAlertFormValues` interface, which is referenced in multiple other places.